src/Microsoft.AspNetCore.Mvc.Razor/Internal/MvcRazorMvcViewOptionsSetup.cs

```diff
@@ -15,32 +15,31 @@ public class MvcRazorMvcViewOptionsSetup : ConfigureOptions<MvcViewOptions>
         /// <summary>
         /// Initializes a new instance of <see cref="MvcRazorMvcViewOptionsSetup"/>.
         /// </summary>
-        /// <param name="serviceProvider">The application's <see cref="IServiceProvider"/>.</param>
-        public MvcRazorMvcViewOptionsSetup(IServiceProvider serviceProvider)
-            : base(options => ConfigureMvc(serviceProvider, options))
+        /// <param name="razorViewEngine">The <see cref="IRazorViewEngine"/>.</param>
+        public MvcRazorMvcViewOptionsSetup(IRazorViewEngine razorViewEngine)
```

Comment:
Would our lives here be simpler if we got rid of the `ConfigureOptions` base class?